### DISCUSSION TITLE
Roborock Qrevo Curv

### DISCUSSION THREAD

**Discussion by choevi** [copystring/ioBroker.roborock#663] [General]
Habe den Roborock Qrevo Curv gerade hinzugefügt, funktioniert auch im wesentlichen. Wenn ich irgendwie Infos für eine volle Unterstützung beitragen kann, sehr gerne.

`Failed to execute undefined on robot undefined (unknown model): This robot roborock.vacuum.a135 is not fully supported just yet. Contact the dev to get this robot fully supported!`

**Comment by copystring** [comment]
Hallo @choevi, bitte in Issue eröffnen und dort ein Debuglog hochladen. Danke!
